src/main/java/com/microsoft/spring/data/gremlin/conversion/source/GremlinSourceEdge.java

```diff
@@ -7,6 +7,7 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import org.springframework.lang.NonNull;
```

Comment:
I don't think it's necessary

Reply:
intellij insert this itself, or it will raise error message.